core/components/organisms/pageHeader/__stories__/withNavigation.story.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import { Navigation, Button, PageHeader } from '@/index';
+import { action } from '@storybook/addon-actions';
+
+export const withNavigation = () => {
+  const navigationData = [
+    {
+      name: 'menu_1',
+      label: 'Virtual Visits',
+    },
+    {
+      name: 'menu_2',
+      label: 'Assesments'
+    }
+  ];
+  const navigation = <Navigation menus={navigationData} onClick={action('menu-clicked')} active={{ name: 'menu_1' }} />;
```

Comment:
Please add a state corresponding to `active` prop so that active navigation menu can update on click.